src/modules/users/graphql/type-query.ts

```diff
@@ -0,0 +1,23 @@
+export interface UserListResponseNodes {
```

Comment:
por ser o tipo do nó eu deixaria sem o plural aqui

```suggestion
export interface UserListResponseNode {
```

Reply:
ah sim, eu deixei como nodes porque como na api retorna nodes, ficava mais facil de associar